Docs update (#514)
elastic/apm-agent-android: docs/reference/automatic-instrumentation.md

2. Add its Gradle plugin to your project in the same location where the [agent](getting-started.md#gradle-setup) is added.
3. [Initialize the agent](getting-started.md#agent-setup) the same way you would without using automatic instrumentation. Automatic instrumentations will get installed during the agent initialization without having to write extra code.

#### OpenTelemetry Android instrumentations

```{applies_to}

> product: beta

Author: ```suggestion
product: technical preview
```

```

You can use instrumentations from [OpenTelemetry Android](https://github.com/open-telemetry/opentelemetry-android/tree/main/instrumentation) with the Elastic agent. Learn how to do so [here](#opentelemetry-android-instrumentation-adapter).

```{important}

> This is an [experimental](component-stability.md#experimental) feature.

Author: ```suggestion
This is a [technical preview](component-stability.md#technical-preview) feature.
```

```

1. You can find the latest version [here](https://plugins.gradle.org/plugin/co.elastic.otel.android.instrumentation.okhttp).

## OpenTelemetry Android Instrumentation Adapter

> 
> **Status**: [experimental](component-stability.md#experimental)
> 

Reviewer: ````suggestion
```{applies_to}
product: experimental
```
````

Author: Can we use the `{applies_to}` approach and still keep the link to the experimental status definition?

Reviewer: I think we can! Let me edit
